src/main/java/core/basesyntax/FigureDraw.java

```diff
@@ -0,0 +1,5 @@
+package core.basesyntax;
+
+public interface FigureDraw {
+    void drawFigure();
```

Comment:
```suggestion
    void draw();
```
We already see the name)